spk/mono/Makefile

```diff
@@ -37,3 +38,9 @@ mono_extra_install:
 	cp -nR $(MONO_NATIVE_INSTALL)/usr/local/etc/mono $(STAGING_DIR)/etc/
 	install -m 755 -d $(STAGING_DIR)/bin
 	cp -nR $(MONO_NATIVE_INSTALL)/usr/local/bin $(STAGING_DIR)/
+
+.PHONY: mono_pre_copy
+mono_pre_copy: $(INSTALL_PLIST)
+	# remove boehm GC references from PLIST if it isn't available on the platform (e.g. rtd1296)
+	test -f $(INSTALL_DIR)$(INSTALL_PREFIX)/bin/mono-boehm || sed -i -e '/:bin\/mono-boehm/d' -e '/:lib\/libmonoboehm/d' $(INSTALL_PLIST)
```

Comment:
Is there any better way to make files from the PLIST "optional"?

Reply:
Not that I'm aware of. The only other option I see would be to disable libboehm and remove it from all packages, but I think your `sed`s are better...